### DISCUSSION TITLE
Github Login - Can't Access Email

### DISCUSSION THREAD

**Discussion by nikhilsharma93** [community/community#88452] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm trying to log in to my work github account and it's sending me a verification code to the work email address that can no longer receive emails (the company switched to a different email provider). I don't have 2FA setup, and I don't see any other way to log in. How can I deal with this situtation? I've already submitted a github support ticket, I just want to see if there's any advice I can get while I wait for them to get back to me.

**Comment by Abrar-Hosny** [comment]
Hope everything will be good with you

**Comment by Abrar-Hosny** [comment]
It's unfortunate that you're facing this issue, but there are a few steps you can take to try to resolve it while waiting for GitHub support to respond:

1. **Check Spam/Junk Folder:**
   Sometimes, verification emails may end up in the spam or junk folder of your email. Make sure to check these folders to see if the email is there.

2. **Contact IT Support:**
   Reach out to your company's IT support or the person responsible for managing GitHub accounts within your organization. They may have alternative methods or processes for handling account access issues.

3. **Verify Email Forwarding:**
   Confirm with your IT department whether there's a forwarding rule set up for your old work email address to redirect emails to your new one. If not, they may be able to set up a temporary forwarding rule for you.

4. **Email Provider Migration:**
   If your company recently switched email providers, inquire about any procedures or changes in place to handle account access during the migration. There may be a temporary solution they can provide.

5. **Check with Colleagues:**
   Consult with colleagues who might have faced a similar situation or have access to the necessary information. They may be aware of any internal procedures for handling these cases.

6. **Temporary Access:**
   If you have a trusted colleague with access to GitHub, ask them to add your account to a repository temporarily. Once you gain access, you can update your email address and regain control of your account.

Remember, these suggestions are general and may not apply to every situation. If your organization has specific policies or procedures for handling account access issues, make sure to follow them. While waiting for GitHub support, regularly check your email for their response and provide any additional information they may request.

**Reply by queenofcorgis** [reply]
Hi @Abrar-Hosny , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by mishmanners** [comment, marked as answer]
For this, you'll need to submit a GitHub Support ticket: https://docs.github.com/en/support/contacting-github-support

**Reply by panasorkokerk1** [reply]
I am in the same situatian. Where exactyle I should put this ticket ? When I click on link and : Creating a support ticket , I don't see option to post a "topic" ?  Where is this, please navigate me ? 
https://github.com/orgs/community/discussions?page=1 
Here or ?

**Reply by seltzdesign** [reply]
Yeah, kind of weird that to post a support ticket, we have to be logged in and then when we choose a topic like Account Issues, it asks to select the account we are having problems with. Well, the account I am having problems with I cant log in to, so how am I supposed to select it if I am not logged in with it!?!?

**Comment by adibbiniqbal** [comment]
If you can’t receive the verification email and you don’t have backup 2FA or recovery methods set up, there’s usually no self-service way to bypass it. Since you already contacted GitHub support, that is the correct step — they are the only ones who can manually verify your identity and update the email. While waiting, try checking if your old work mailbox still has forwarding enabled, or ask your former IT/admin team if they can temporarily restore access, but otherwise you’ll need to wait for support to resolve it.
